src/amo/components/WrongPlatformWarning/index.js

```diff
@@ -35,46 +43,61 @@ type InternalProps = {|
 export class WrongPlatformWarningBase extends React.Component<InternalProps> {
   static defaultProps = {
     _correctedLocationForPlatform: correctedLocationForPlatform,
+    _isFenix: isFenix,
   };
 
   render() {
     const {
       _correctedLocationForPlatform,
+      _isFenix,
       className,
       clientApp,
       i18n,
       location,
       userAgentInfo,
     } = this.props;
 
-    const newLocation = _correctedLocationForPlatform({
-      clientApp,
-      location,
-      userAgentInfo,
-    });
+    let message;
 
-    if (!newLocation) {
-      return null;
-    }
+    if (_isFenix(userAgentInfo)) {
+      message = i18n.sprintf(
+        this.props.fixFenixLinkMessage ||
+          i18n.gettext(
+            `To find add-ons compatible with Firefox on Android,
```

Comment:
I think it is `Firefox for Android`

Reply:
Indeed! That's also what's in the doc at https://docs.google.com/document/d/1HOImPu2RO9PXIbrNvE_YhYyONw7n-rgjzMtMQ3Knk4Q/edit?ts=5e18f98a#heading=h.lzrviwt2ql63. I have no idea where that "on" came from!